README.md

```diff
@@ -20,6 +20,25 @@ SlowMovie is the code that runs a VSMP on a Raspberry Pi.
 
 SlowMovie requires [Python 3](https://www.python.org). It uses [FFmpeg](https://ffmpeg.org) via [ffmpeg-python](https://pypi.org/project/ffmpeg-python) for video processing, and [Pillow](https://python-pillow.org) for image processing. [ConfigArgParse](https://pypi.org/project/ConfigArgParse) is used for configuration and argument handling.
 
+### Quick install
+
+You can quickly install this repository and all required libraries via an install script. This is a simple way to get started if you're not as comfortable with the command line. You'll be presented with 4 options when you run the script:
+
+1. Install/Upgrade SlowMovie - this will download the repository and install/update any libraries needed
+2. Install SlowMovie Service - run the commands install the SlowMovie service file as described below
+3. Uninstall SlowMovie Service - uninstall the SlowMovie service
+4. Exit
+
+```
+
+bash <(curl https://raw.githubusercontent.com/TomWhitwell/SlowMovie/master/Install/install.sh)
+
+```
+
+You can re-run the script at any time in the `Install` folder to update.
+
+### Manually
```

Comment:
Just some tweaks to make this a bit clearer hopefully.

Note that I couldn't use the triple-backtick format for code blocks while doing one of these review suggestions, so I had to change it to the quadruple-spaces formatting. It makes the same thing when rendered but it would be nice to keep it looking like
\```
bash .....
\```
In the actual change for consistency with the rest of the readme.

```suggestion
### Automated installation

You can quickly install this repository and all required libraries via an install script. This is a simple way to get started if you're not as comfortable with the command line.

To run the install script, open your terminal, copy-paste the following command in, and hit enter.

    bash <(curl https://raw.githubusercontent.com/TomWhitwell/SlowMovie/master/Install/install.sh)

You'll be presented with 4 options when you run the script:

1. **Install/Upgrade SlowMovie** - download the repository and install/update any libraries needed
2. **Install SlowMovie Service** - run the commands install the SlowMovie service file as described below
3. **Uninstall SlowMovie Service** - uninstall the SlowMovie service
4. **Exit**

You can re-run the script at any time in the `Install` folder to update.

### Manual installation
```